src/blockchain/extrinsic.ts

```diff
@@ -70,16 +70,11 @@ export class Extrinsic<T extends ISubmittableResult = ISubmittableResult, C exte
     return firstValueFrom(
       this.extrinsic.signAndSend(this.keys, { nonce }).pipe(
         timeout({ each: this.timeOutSeconds * 1000 }),
-        tap(({ status }) => {
-          if (!status.isInBlock || !status.isFinalized) {
-            this.signAndSend(nonce);
```

Comment:
I think this might have been the cause of the crash. I think perhaps we were overloading the transaction pool with so many duplicate-nonce transactions, it was causing a chain disconnection right in this subscription stream and crashing the app.